### ISSUE TITLE
How do I designate the ±/§ key found on some European keyboards?

### ISSUE THREAD

**Issue by pmetzger** [pqrs-org/Karabiner-Elements#2841]
Say I want to remap the Caps Lock key (in a "simple" modification) to the key found on a UK or other European keyboard that is "§" when unshifted ("±" when shifted). The list of options in the provided list appears not to include this key? Also, if I wanted to designate that keycode in a complex modification, how would I do it?

**Comment by pmetzger** [comment]
Partially answering my own question: the pulldown appears to refer to this as "non_us_backslash" even though the keycode is named kVK_ISO_Section and in most cases produces the "§" character. Is it possible to either document this or to provide an alias named "section"?

**Comment by MuhammedZakir** [comment]
All of names used for keycodes (except sticky keys) are names from Apple [1]. You can find the `keycode` of a key using EventViewer [2].

[1] https://opensource.apple.com/source/IOHIDFamily/IOHIDFamily-315.7.16/IOHIDFamily/IOHIDUsageTables.h
[2] `Karabiner-EventViewer.app`, or `Karabiner-Elements menu > Launch EventViewer`.

**Comment by pmetzger** [comment]
Yes, I'm aware of that, but the key is a § on most of the Europe's keyboards and most people aren't going to think of it as "non-US backslash". An alias might be nice. At the very least, it would make the program slightly more user friendly.

**Comment by pmetzger** [comment]
Just to be clear, the apple keycode for this key in HIToolbox/Events.h is:

```
/* ISO keyboards only*/
enum {
  kVK_ISO_Section               = 0x0A
};
```

i.e. "Section"

**Comment by tigger04** [comment]
I have seen this raised a few times over the years. It's a pretty clear bug in Karabiner with a pretty clear solution but for some reason the developers seem intent to double down on this mistake

**Comment by tigger04** [comment]
At the very least it would seem important to document it, since it is such a common key and the question seems to come up a lot.

**Comment by pmetzger** [comment]
So the bot closes issues that no one works on even if they're not fixed?

**Comment by tigger-developer** [comment]
In modern karabiner it is (oddly) named `non_us_backslash` - works for me anyway

**Comment by pmetzger** [comment]
I know it's been five years but it would still be nice if there was an alias for this that normal people might understand.

**Comment by tekezo** [comment]
It is difficult to handle automatically because of how macOS behaves. However, the beta version now lets you swap ISO layout-specific keys under Devices, so please give it a try.

How to upgrade Karabiner-Elements to the latest beta version:
<https://karabiner-elements.pqrs.org/docs/manual/operation/check-for-updates/#using-the-latest-beta-version>

<img width="1100" height="732" alt="Image" src="https://github.com/user-attachments/assets/ebba5fbb-d9e6-4505-aa84-6780e8261384" />

**Comment by pmetzger** [comment]
Couldn't the key get two names?
